e2e/scripts/Dockerfile

```diff
@@ -0,0 +1,7 @@
+#Use and official Python runtime as a base image
+FROM quay.io/influxdb/influx:nightly
```

Comment:
what's all this gobbledygook?

Reply:
that's influxdb 2.0.. lol, and copypasta